### DISCUSSION TITLE
antv/g6 v4最新版本兼容v3.0.5版本吗

### DISCUSSION THREAD

**Discussion by xiao-2** [antvis/G6#3651] [Q&A]

**Comment by gotounix** [comment]
不兼容，我从3.7.3升级到4.1.0都是一场灾难。
CHANGELOG太简洁了，一堆报错无从下手。
建议，没有大需求，别升级。

**Reply by xiao-2** [reply]
好的  非常感谢

**Comment by Yanyan-Wang** [comment]
升级指南：

https://g6.antv.vision/zh/docs/manual/FAQ/upgradeGuide
https://github.com/antvis/G6/issues/2525
